install.ps1

```
@@ -14,35 +14,80 @@ if ($Arch -ne "AMD64") {
-            return $versionInfo.version
+            # The version endpoint can advertise a release before its binaries
+            # are uploaded (or one whose release pipeline failed). Trust it
+            # only when the platform asset is actually downloadable.
+            if (Test-AssetExists $versionInfo.version) {
+                return $versionInfo.version
+            }
+            Write-Warning "$($versionInfo.version) is advertised but its release assets are not available yet (release may be in progress). Falling back to the newest installable release..."
         }
     } catch {
         # Fall through to GitHub API
     }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)